xsec/plotting_functions.h

```diff
@@ -116,38 +116,42 @@ class EventSelectionPlotInfo {
 void SetErrorGroups(MnvPlotter& mnv_plotter) {
```

Comment:
These changes are not related to Adler angle stuff.

It's obviously not a big deal, but branch X should only contain changes related to X.

These changes, for example, could go directly into master.